.travis.yml

```diff
@@ -1,22 +1,41 @@
 services:
-- docker
+  - docker
+
+branches:
+  only:
+  - exercise-8
+
 script:
-- docker build --tag todo-app-dev --target development .
-- docker build --tag todo-app-test --target test .
-- docker build --tag todo-app-prod --target production  .
-- docker run -e MY_SECRET -e SERVER_TOKEN -e DEVELOPER_API_KEY todo-app-test:latest
+  - #docker build --tag todo-app-dev --target development .
+  - #docker build --tag todo-app-test --target test .
+  - docker build --tag $DOCKER_USER/todo-app-prod:$TRAVIS_COMMIT --target production  .
+  - #docker run -e MY_SECRET -e SERVER_TOKEN -e DEVELOPER_API_KEY -e FLASK_APP -e FLASK_ENV todo-app-test:latest
+  - echo "Push Production image to Docker Hub"
+  - echo $DOCKER_PWD
```

Comment:
You're echoing out your password to the console, for anyone to see! We should remove this, and rotate your Dockerhub password (and any other service that uses the same!)